src/screens/Gurbani/BottomBar.tsx

```diff
@@ -47,7 +49,12 @@ const BottomBar = () => {
 
   return (
     <SafeAreaView style={styles.root} edges={[ 'bottom', 'left', 'right' ]}>
-      <View style={styles.background} />
+
+      <LinearGradient
+        style={styles.background}
+        locations={[ 0.0, 0.733 ]}
+        colors={gradient.TransparentToBlack.reverse()}
```

Comment:
```suggestion
        colors=[{[...gradient.TransparentToBlack].reverse()}
```

as `.reverse()` runs in place. This means that every render, it is possible for the gradient to switch direction 😄